test.sh

```diff
@@ -100,8 +100,10 @@ function FindCustomClusterTests {
 
 function FindDefaultClusterTests {
     touch $DEFAULT_CLUSTER_TESTS
-    for PKG in $(grep -v -f $CUSTOM_CLUSTER_TESTS $MATCHING_TESTS); do
```

Comment:
I think grep -v -f works slightly differently on a mac, so I was always getting no tests matched whenever i passed in a matcher (like 'query').